crmd/remote_lrmd_ra.c

```diff
@@ -662,9 +685,14 @@ handle_remote_ra_stop(lrm_state_t * lrm_state, remote_ra_cmd_t * cmd)
     }
     ra_data->cmds = NULL;
     ra_data->recurring_cmds = NULL;
-    ra_data->cur_cmd = NULL;
 
-    report_remote_ra_result(cmd);
+    if (cmd) {
+        cmd->rc = PCMK_OCF_OK;
+        cmd->op_status = PCMK_LRM_OP_DONE;
+        ra_data->cur_cmd = NULL;
```

Comment:
didn't want to argue against moving 'ra_data->cur_cmd = NULL' just wanted to give
an explanation why I hadn't cared much about thinking about what happens if there
is a cmd on the way.